datadog_checks_dev/datadog_checks/dev/tooling/commands/test.py

```diff
@@ -147,6 +147,8 @@ def test(
         test_env_vars['TOX_TESTENV_PASSENV'] += ' TF_BUILD BUILD* SYSTEM*'
         test_env_vars['DD_SERVICE'] = os.getenv('DD_SERVICE', 'ddev-integrations')
         test_env_vars['DD_ENV'] = os.getenv('DD_ENV', 'ddev-integrations')
+        test_env_vars['DDEV_TRACE_ENABLED'] = 'true'
+        test_env_vars['DD_PROFILING_ENABLED'] = 'true'
```

Comment:
Not too familiar with dd_trace, but what is the purpose of enabling `DD_PROFILING_ENABLED` for this?

Reply:
The purpose is to enable the profiling product as well (https://ddtrace.readthedocs.io/en/stable/configuration.html) so we have even more info on the behaviour on our integrations https://docs.datadoghq.com/tracing/profiler/